tmp/axum-wasm-expanded/src/lib.rs

```diff
@@ -44,15 +48,25 @@ async fn uppercase(body: BodyStream) -> impl IntoResponse {
 #[no_mangle]
 #[allow(non_snake_case)]
 pub extern "C" fn __SHUTTLE_Axum_call(
+    fd_2: std::os::wasi::prelude::RawFd,
     fd_3: std::os::wasi::prelude::RawFd,
     fd_4: std::os::wasi::prelude::RawFd,
     fd_5: std::os::wasi::prelude::RawFd,
 ) {
     use axum::body::HttpBody;
+    use shuttle_common::wasm::Logger;
     use std::io::{Read, Write};
     use std::os::wasi::io::FromRawFd;
+    use tracing_subscriber::prelude::*;
 
-    println!("inner handler awoken; interacting with fd={fd_3},{fd_4},{fd_5}");
+    println!("inner handler awoken; interacting with fd={fd_2},{fd_3},{fd_4},{fd_5}");
```

Comment:
Should we change these to `trace!`? Here and in the codegen.

Reply:
I wanted to initially, but ended up deciding not to for this reason: if a user decides not to use a `tracing` call in a request path then the request has our current 10ms latency. But adding a `tracing` call makes the latency 12ms. So I figured it best to safe the 2ms for users who don't use `tracing` :smile: